### PR TITLE
bump to v2.1.0

### PR DESCRIPTION
**Pull request by bjlittle**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 package:
-    name: iris-sample-data
+    name: {{ name }}
     version: {{ version }}
 
 source:
-    fn: iris-sample-data-{{ version }}.tar.gz
-    url: https://github.com/SciTools/iris-sample-data/archive/v{{ version }}.tar.gz
-    sha256: 8907f2ac0e38a32e39f79db7e6426d25c27cb18b9ad510a33f3f2538ebac12ac
+    fn: {{ name }}-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml]: There is no need for `fn`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 
 package:
-    name: iris-sample-data
+    name: {{ name }}
     version: {{ version }}
 
 source:
-    fn: iris-sample-data-{{ version }}.tar.gz
-    url: https://github.com/SciTools/iris-sample-data/archive/v{{ version }}.tar.gz
-    sha256: 8907f2ac0e38a32e39f79db7e6426d25c27cb18b9ad510a33f3f2538ebac12ac
```
Comment (ocefpaf) [inline recipe/meta.yaml]: The checksum here will make it easier to review when the bot sends the PRs.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -25,10 +27,12 @@ test:
         - iris_sample_data
 
 about:
-    home: https://github.com/SciTools/iris-sample-data
+    home: https://github.com/SciTools/{{ name }}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Using a jinja var for name makes the recipe hard to read.

**Comment by ocefpaf** [comment]
@conda-forge-admin, please rerender

**Comment by ocefpaf** [comment]
@conda-forge-admin, please add noarch: python

**Comment by bjlittle** [comment]
@ocefpaf Thanks for pushing this recipe forwards. Much appreciated!

**Comment by ocefpaf** [comment]
> @ocefpaf Thanks for pushing this recipe forwards. Much appreciated!

No problem.

PS: with `noarch` we can merge this faster b/c the recipe is built only once.

**Comment by bjlittle** [comment]
Yeah, this make sense since it's pure data.

I was holding back on making it a noarch package as its a minor internal problem for us here, but I can patch that out at our end until we fix that problem (our problem). 

You're right to push the noarch now. Faster is better (generally).

**Comment by ocefpaf** [comment]
We can revert the `noarch` if that is simpler.

**Comment by bjlittle** [comment]
No, it's totally fine @ocefpaf ... merge away!

**Comment by bjlittle** [comment]
Awesome sauce. Thanks @ocefpaf !

**Pull request merged**